Update monitoring-partners.md

rbormann opened the pull request:
Hello,

I wanted to suggest an update to this article for including the SIGNL4 alert notification service for AM.
https://www.signl4.com/blog/mobile-alert-notifications-azure-monitor/
https://www.signl4.com/

I have however no repository permission to upload a logo.

Best regards,
René

azurecla commented:
Hi __@chaosrealm__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

rbormann committed:
Update monitoring-partners.md

Hello,

I wanted to suggest an update to this article for including the SIGNL4 alert notification service for AM.
https://www.signl4.com/blog/mobile-alert-notifications-azure-monitor/
https://www.signl4.com/

I have however no repository permission to upload a logo.

Best regards,
René
